GDAXClient/Services/Products/ProductsService.cs

```diff
@@ -68,5 +68,13 @@ public class ProductsService : AbstractService
 
             return productStatsResponse;
         }
+
+        public async Task<IList<IList<ProductTrades>>> GetProductTradesAsync(ProductType productPair, int limit = 100)
+        {
+            var trades = await SendHttpRequestMessagePagedAsync<ProductTrades>(HttpMethod.Get,
+                authenticator, $"/products/{productPair.ToDasherizedUpper()}/trades");
```

Comment:
you need to add the limit as query parameter.

there is a query builder for this now if you want to follow an example: https://github.com/dougdellolio/gdax-csharp/pull/30/files#diff-16a6f44920b852dba9b8cca55bcb422eR37